cli/share_create_test.go

```diff
@@ -15,9 +20,19 @@ func TestValidateTag(t *testing.T) {
 	for _, tc := range cases {
 		got := validateTag(tc.tag)
 		valid := got == nil
+		assert.Equal(t, tc.valid, valid, "validateTag(%s) == nil should be %t but got %s", tc.tag, tc.valid, got)
+	}
+}
 
-		if valid != tc.valid {
-			t.Errorf("validateTag(%s) == nil should be %t but got %s", tc.tag, tc.valid, got)
-		}
+func TestGetDefaultStepGroupSpec(t *testing.T) {
+	// Given
+	expected := models.StepGroupInfoModel{
+		Maintainer: "community",
 	}
+
+	// When
+	actual := getDefaultStepGroupSpec()
+
+	// Then
+	assert.Equal(t, expected, actual)
```

Comment:
I would suggest using `require.*` instead of `assert.*`. The difference is, `require` fails if the condition is not met, while `assert` only returns a bool. https://github.com/stretchr/testify#require-package

Reply:
And so `assert` will not even fail the test? 😮

Reply:
This is my understanding based on the documentation.

Reply:
I did not check it though ...

Reply:
I just tried it, and it fails the test.
I would go with `assert`, because that naming is widely adopted already in testing contexts, and `require` is usually used for input argument validation. 
The additional feature of stopping the test is not required because the assertions are independent, and for example in `Test_GivenHomeDir_WhenGetStepmanDirPathCalled_ThenGoodPathReturned` I use `require` because if the setup fails, that should indeed stop the test.

So in general I'd vote for `assert`

Reply:
assert is fine